tools/ci_build/github/azure-pipelines/templates/linux-web-ci.yml

```diff
@@ -0,0 +1,176 @@
+    condition: ne('${{ parameters.CommitOverride }}', 'true')
+  - script: |
+     git submodule sync -- cmake/external/onnx
+     git submodule update --init -- cmake/external/onnx
+    workingDirectory: '$(Build.SourcesDirectory)'
+    displayName: 'Checkout submodule onnx'
+  - script: |
+     echo.>>.gitattributes
+     echo /js/** text=auto eol=lf>>.gitattributes
+     rm -rf js
+     git checkout -- js/**
+     git checkout -- .gitattributes
+    workingDirectory: '$(Build.SourcesDirectory)'
+    displayName: 'Testing: force EOL to lf on windows for /js/**'
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '16.x'
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      patterns: '${{ parameters.BuildConfig }}_*/**/*'
+      path: $(Pipeline.Workspace)/artifacts
+    displayName: 'Download WebAssembly artifacts'
+  - task: CopyFiles@2
+    inputs:
+      sourceFolder: $(Pipeline.Workspace)/artifacts
+      contents: |
+        **/*.wasm
+      targetFolder: $(Build.SourcesDirectory)/js/web/dist
+      flattenFolders: true
+    displayName: 'Binplace dist files'
+  - task: CopyFiles@2
+    inputs:
+      sourceFolder: $(Pipeline.Workspace)/artifacts
+      contents: |
+        **/*.js
+      targetFolder: $(Build.SourcesDirectory)/js/web/lib/wasm/binding
+      flattenFolders: true
+    displayName: 'Binplace js files'
+  - script: |
+     npm ci
+    workingDirectory: '$(Build.SourcesDirectory)/js'
+    displayName: 'npm ci /js/'
+  - script: |
+     npm run lint
+    workingDirectory: '$(Build.SourcesDirectory)/js'
+    displayName: 'run ESLint without TS type populated'
+  - script: |
+     npm ci
+    workingDirectory: '$(Build.SourcesDirectory)/js/common'
+    displayName: 'npm ci /js/common/'
+  - script: |
+     npm ci
+    workingDirectory: '$(Build.SourcesDirectory)/js/web'
+    displayName: 'npm ci /js/web/'
+  - script: |
+     npm run lint
+    workingDirectory: '$(Build.SourcesDirectory)/js'
+    displayName: 'run ESLint'
+  - script: |
+     npm run format
+    workingDirectory: '$(Build.SourcesDirectory)/js'
+    displayName: 'Clang-format'
+  - script: |
+     node -e "a=require('child_process').execSync('git diff --name-only').toString();if(a)throw new Error('Following source files are not formatted: (did you run \"npm run format\"?)\n'+a)"
+    workingDirectory: '$(Build.SourcesDirectory)/js'
+    displayName: 'Check unformatted files'
+  - script: |
+     npm run build:doc
+    workingDirectory: '$(Build.SourcesDirectory)/js/web'
+    displayName: 'Generating documents'
+  - script: |
+     node -e "a=require('child_process').execSync('git diff --name-only').toString();if(a)throw new Error('Following documents are not up-to-date: (did you run \"npm run build:doc\"?)\n'+a)"
+    workingDirectory: '$(Build.SourcesDirectory)/js/web'
+    displayName: 'Check out of dated documents'
+  - task: PowerShell@2
+    inputs:
+      filePath: '$(Build.SourcesDirectory)/tools/ci_build/github/js/pack-npm-packages.ps1'
```

Comment:
in file `pack-npm-packages.ps1`, there are a few lines that using `Join-Path` with windows path split '\', for example:

```ps
$JS_COMMON_DIR=Join-Path -Path "$ORT_ROOT" -ChildPath "js\common"
$JS_TARGET_DIR=Join-Path -Path "$ORT_ROOT" -ChildPath "js\$TARGET"
```

those may not work correctly on linux if not update

Reply:
thx, updated